### PR TITLE
virttest.qemu_devices: Request pci slot for pci-* serial devs

### PR DESCRIPTION
**Pull request by ldoktor**
There are pci-serial and pci-serial-4x devices that are plugged into a
PCI slot. Let's assume all serial devices with "pci-" prefix require PCI
and request one.

Signed-off-by: Lukáš Doktor <ldoktor@redhat.com>

### CODE REVIEW

File: virttest/qemu_devices/qcontainer.py [hunk 1]
```diff
@@ -2259,7 +2259,12 @@ def serials_define_by_variables(self, serial_id, serial_type, chardev_id,
                                  parent_bus={'busid': bus}))
             devices[-1].set_param('name', serial_name)
         else:  # none virtio type, generate serial device directly
-            devices.append(qdevices.QDevice(serial_type, {"id": serial_id}))
+            if serial_type.startswith("pci"):
```
Comment (xutian) [inline virttest/qemu_devices/qcontainer.py:2262]: If machine type is "Q35",  pci-serial will plug  under root port. is it expect ?

Comment (xutian) [reply virttest/qemu_devices/qcontainer.py:2262]: > If machine type is "Q35", pci-serial will plug under root port. is it expect ?

@jingzhao84  can you have a try ?

**Comment by yanan-fu** [comment]
Ah~ i did not notice this patch before. I submitted a new one few days ago:
https://github.com/avocado-framework/avocado-vt/pull/2566

It plug the pci serial device to the parant bus 'pci' under pc machine type, and plug it to parent bus 'pci-bridge' under machine type q35.

**Comment by yanan-fu** [comment]
But, i only consider `pci-serial`, did not cover `pci-serial-4x`.

**Comment by ldoktor** [comment]
Closing in favor of #2571

**Pull request closed without merging**